### PR TITLE
update homer to new module structure

### PR DESCRIPTION
**Pull request by alexthiery**
just updating to new module structure - passing args and meta etc

### CODE REVIEW

File: configs/module-defaults.config [hunk 1]
```diff
@@ -138,5 +138,11 @@ params {
             publish_dir      = "htseq_count"
             publish_results  = "all"
         }
+        'homer_annotatePeaks' {
```
Comment (chris-cheshire) [inline configs/module-defaults.config:141]: no camel case here - just underscores

File: tools/homer/main.nf [hunk 2]
```diff
@@ -5,24 +5,44 @@ nextflow.enable.dsl=2
...
+        args = ""
+        if(opts.args && opts.args != '') {
+            ext_args = opts.args
+            args += ext_args.trim()
+        }
+
+        prefix = opts.suffix ? "${meta.sample_id}${opts.suffix}" : "${meta.sample_id}"
+
+    // Construct CL line
+    annotatePeaks_command = "annotatePeaks.pl ${args} ${peaks} ${fasta} -gtf ${gtf} > ${prefix}"
```
Comment (chris-cheshire) [inline tools/homer/main.nf:36]: again, mix of camel and underscores

File: tools/homer/main.nf [hunk 3]
```diff
@@ -5,24 +5,44 @@ nextflow.enable.dsl=2
 
 //Process definition
 process homer_annotatePeaks {
```
Comment (chris-cheshire) [inline tools/homer/main.nf:7]: naming convention

Comment (alexthiery) [reply tools/homer/main.nf]: my bad. fixed.

**Comment by alexthiery** [comment]
not sure why it is expecting line-docker for homer. I have removed this from the workflows as I have switched to the biocontainer

**Pull request merged**